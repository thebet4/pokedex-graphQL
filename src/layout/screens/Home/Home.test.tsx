// import * as React from "react";
// import renderer from "react-test-renderer";

// import Home from "@layout/screens/Home";
// import {
//   NativeStackNavigationProp,
//   NativeStackScreenProps,
// } from "@react-navigation/native-stack";
// import { AppStackParamList } from "@routes/AppRoutes";

// it(`renders <Home /> correctly`, () => {
//   let navigation: any;
//   let route: any;

//   const tree = renderer
//     .create(<Home navigation={navigation} route={route} />)
//     .toJSON();

//   expect(tree).toMatchSnapshot();
// });
