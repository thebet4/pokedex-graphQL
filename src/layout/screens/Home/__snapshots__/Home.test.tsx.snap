// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <Home /> correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#FFFFFF",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
      },
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "backgroundColor": "#21386E",
          "borderBottomLeftRadius": 15,
          "borderBottomRightRadius": 15,
          "height": 75,
          "justifyContent": "center",
          "width": "100%",
        },
      ]
    }
  >
    <Image
      source={1}
    />
  </View>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#FFFFFF",
          "flexBasis": 0,
          "flexGrow": 1,
          "flexShrink": 1,
          "marginLeft": 15,
          "marginRight": 15,
          "marginTop": 20,
        },
      ]
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#21386E",
            "fontFamily": "Inter_700Bold",
            "fontSize": 20,
            "letterSpacing": 0,
            "lineHeight": 24,
            "textAlign": "left",
          },
        ]
      }
    >
      More than 250 Pokemons for you to choose your favorite
    </Text>
  </View>
</View>
`;
